src/Avalonia.Base/Threading/DispatcherPriority.cs

```diff
@@ -48,20 +44,26 @@ public enum DispatcherPriority
         /// The job will be processed with the same priority as render.
         /// </summary>
         Render = 7,
-
+        
+        /// <summary>
+        /// The job will be processed with the same priority as render.
+        /// </summary>
+        Layout = 8,
```

Comment:
The `Layout` enum comes from WPF which doesn't have a `Layout` priority. Why is it necessary here but not in WPF?

Reply:
I'm not really familiar with WPF internals, but I think that `LayoutManager::ExecuteLayoutPass` should always happen before `DeferredRenderer` grabs a scene update and `DataBind` isn't exactly meant for layout.